MODEL-GUIDE.md

```diff
@@ -302,6 +303,79 @@ The build script will enforce the following keyword conventions:
+    * The bundler will populate `enum` at the time of bundling. So, it will follow OpenAPI standard. 
+    * `x-enum` remain within final yaml file to generate unique proto uid.
+
+    <img src="images/use_case_2.PNG" alt="drawing" width="500"/>
+
+  * Use-Case : Including a Pre-Existing Object
+    * `x-include` to include only the field of the property (`..#/components/schemas/GlobalObject/properties/name`)
+    * Same property name (`uniquename`) must be used for both base and derived property. 
+    * The bundler will correctly merge the `x-include` (e.g. `type` and `pattern`) and drop the `x-include` keyword from the merged object.
+    * At the time of merge it will not include base `x-field-uid` `x-field-uid: 2` should present in the derived property.
+    
+    <img src="images/use_case_3.PNG" alt="drawing" width="500"/>
+
+  * Use-Case : Deprecating a Property 
+    * Model should maintain `x-field-uid: 1` when property will deprecate 
+    * `x-status` must be added with status deprecated
+
+    <img src="images/use_case_4.PNG" alt="drawing" width="500"/>
+
+  * Use-Case : Removing a Property
+    * User must never use the numbers added in `x-reserved-field-uids` again.
+    * That `x-field-uid` number should include within existing or new `x-reserved-field-uids`
+    * Try to remove `ieee_802_1qbb` which have `x-field-uid: 1`. That number must be included within `x-reserved-field-uids: [1]`
+
+    <img src="images/use_case_5.PNG" alt="drawing" width="500"/>
+
+  * Use-Case : Removing an enum value
+    * User must never use the numbers added in `x-reserved-field-uids` again.
+    * That `x-field-uid` number should include within existing or new `x-reserved-field-uids`
+    * Removing property `b` which have `x-field-uid: 2`. That number must be included within `x-reserved-field-uids: [2]`
+  
+    <img src="images/use_case_6.PNG" alt="drawing" width="500"/>
+
+  * Use-Case : Removing a Pre-Existing Object
+    * Remove the exiting 'Prefix.Config' object.
+    * All the properties and field within that object must be removed.
+
+    <img src="images/use_case_7.PNG" alt="drawing" width="500"/>
+
+  * Use-Case : Adding / Removing a Response Field 
+    * Adding a new response `400` in the exiting responses. New unique `x-field-uid: 3` is assigned.
+    * User must not use the field UID present within `x-reserved-field-uids: [2]`
+    * That reserved field UID (e.g. 2) used by other property which was removed.
+
+    <img src="images/use_case_8.PNG" alt="drawing" width="500"/>
+
+  * x-reserved-field-uids
+
+    * When a property/response is removed, the `x-field-uid` assigned to that property/response is included in the list of `x-reserved-field-uids` so that those values are never ever re-used.
+    * Use-Case : Removing a Property
+      * User must never use the numbers added in `x-reserved-field-uids` again.
+      * That `x-field-uid` number should include within existing or new `x-reserved-field-uids`
```

Comment:
I didn't understand what we're trying to say here.

Reply:
remove this section